pgmigrate.py

```diff
@@ -219,7 +219,11 @@ def _get_info(base_dir, baseline_v, target_v, cursor):
             if num not in ret:
                 ret[num] = migrations_info[version]
 
-    return ret
+    orderedret = OrderedDict()
+    for version in sorted(ret, key=int):
+        orderedret[version] = ret[version]
+
+    return orderedret
```

Comment:
May be we just need to sort state just before output? Sorted migrations info is not required in other functions.